test/patterns/gen-ai/aws-qa-appsync-opensearch/aws-qa-appsync-opensearch-serverless.test.ts

```diff
@@ -55,6 +55,7 @@ describe('QA Appsync Open search construct', () => {
       openSearchIndexName: 'demoindex',
       openSearchSecret: osSecret,
       cognitoUserPool: userPoolLoaded,
+      sagemakerEndpointName: 'sageMakerEndpoint',
```

Comment:
as mentioned above, I would not add this prop

Reply:
done -7d4d3a152fa0f0b4a28117814347191758cae246